### ISSUE TITLE
Hello author, may I ask if the experimental results of the mandibular canal in the paper are all from the test set? Or is it the result of the training set?

### ISSUE THREAD

**Issue by lgh010319** [mahmoudgamal0/Pulpy3D#4]

**Comment by mahmoudgamal0** [comment]
Hello,

All results are on the test set

**Comment by lgh010319** [comment]
Thank you, author
I have another question, have you ever encountered memory explosion during training and prediction? I noticed that your memory size is 32g, while I am currently using 64g. However, why am I experiencing a memory explosion issue where the program freezes due to a full memory

**Comment by mahmoudgamal0** [comment]
Yes, this can happen if you did preprocessing with resampling and setting the parameter for example to 0.3 while the samples we are providing in pulpy3D are all isotropic set to 1.0. Let me know if this is the case. Since all samples are isotropic you don't need the resampling unless you are training differently than the method suggested in the codebase

**Comment by lgh010319** [comment]
It's not like that, sorry, I didn't express myself clearly. I used my own dataset, but the format of my dataset is from (188，512，512) to (982，1024，1024). Did this generate too many fragments, which caused a memory explosion?

**Comment by mahmoudgamal0** [comment]
Yes, definitely that can be a direct cause

**Comment by lgh010319** [comment]
![image](https://github.com/user-attachments/assets/4eaafce4-6890-4784-802c-44c98f327135)
Okay, author. For example, what software did you use to generate the photos in the paper? I am unable to generate 3D images with both samples and labels using ITKsnap

**Comment by mahmoudgamal0** [comment]
I used Paraview with the NiftiImageReader Addon that you can enable from the desktop application itself

**Comment by lgh010319** [comment]
Hello author, I noticed a question. Why are the labels of the mandibular canal sometimes thick and sometimes thin? For example, 535 is very thin, 504 is very thick, and it is the same as the training set. When running the training set for prediction, 504 performs well, but 535 performs poorlyHello author, I noticed a question. Why are the labels of the mandibular canal sometimes thick and sometimes thin? For example, 535 is very thin, 504 is very thick, and it is the same as the training set. When running the training set for prediction, 504 performs well, but 535 performs poorly
![image](https://github.com/user-attachments/assets/b0b1927f-9581-4b26-aad5-6592617caba8)
![image](https://github.com/user-attachments/assets/e155fcb1-846c-43c5-b7ef-d46ce4778a3e)

**Comment by mahmoudgamal0** [comment]
I think it is a mistake in the label from our side let me check if we uploaded the wrong label.
Let me know if you find other instances

**Comment by lgh010319** [comment]
It's like this, author. I found that the numbers after 520 are all incorrect

**Comment by lgh010319** [comment]
Also, there are these labels in the picture below
![image](https://github.com/user-attachments/assets/76aa1617-3064-4ccd-b8e6-de6d1509faa6)

**Comment by mahmoudgamal0** [comment]
Okay thank you, will trace them and upload the correct ones shortly

**Comment by lgh010319** [comment]
Understood. If you have completed the upload, please reply to me. Thank you

**Comment by lgh010319** [comment]
> 我将 Paraview 与 NiftiImageReader 插件一起使用，您可以从桌面应用程序本身启用该插件

I downloaded this software based on your suggestion, but I noticed that it can only open files in nii format and not nii.gz format. Did you convert nii.gz format to nii format and then open them

**Comment by lgh010319** [comment]
> 好的，谢谢你，我们会尽快追踪他们并上传正确的

Hello author, may I ask if the new dataset has been uploaded?

**Comment by mahmoudgamal0** [comment]
Hello, Yes you can convert the .nii.gz to .nii. by just extracting them. The paraview reader doesn't understand the .gz part.
I am sorry for the delay, we are working on the dataset and will inform you when the new files are uploaded as soon as possible

**Comment by lgh010319** [comment]
> 您好，是的，您可以将 .nii.gz 转换为 .nii。只需提取它们。paraview 读者不理解.gz部分。很抱歉耽误了时间，我们正在处理数据集，并在上传新文件时尽快通知您

I understand. I hope to upload the dataset as soon as possible. Thank you.

**Comment by mahmoudgamal0** [comment]
Sorry for the delay
I have found nothing wrong with 20 and 442
You can find the updated samples in this [link](https://storage.googleapis.com/pulp-temp/updated.zip?x-goog-signature=85508964288bac0112e311b082ad2720ef3ae05fce602a7ac1fc74c001ecd38a24652cac0c8f3f61d61fead24856178c0941f008d9900398234e087da9d51e16e3244505c0087b3aa3baac11ae928ccb7f3a0b8f65c02d6c2a6b5f03d88c4ff29404e015a06e4c48797465bf546bd3029a6783a008b0dc52dca2c1d1fd9d33cf10239a697d48cefb241ba6f0b22e133918aed75db3b8e6aa79fdef1bb3a8345b88e96fdcbb0828c6e70634f5efce1fd102100cd6cf6cee4247c933b74e444a46bcdf8c8f973cf1a5f8cccd27ac92bfdb4a9ceb5d366f051e8b67f9a7ae5f00390bf90561490821ea00fdbc9967b4905e5205095360581de89089fe0bc7904738&x-goog-algorithm=GOOG4-RSA-SHA256&x-goog-credential=public-download%40robotic-center-382715.iam.gserviceaccount.com%2F20241212%2Fus%2Fstorage%2Fgoog4_request&x-goog-date=20241212T065450Z&x-goog-expires=36000&x-goog-signedheaders=host)
This link is available for 10 hours only, let me know if you have any problems
The changes will be reflected on the public channels on a later date, I thought to share the updates with you to save time

**Comment by lgh010319** [comment]
Received, thank you very much
